lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc

```diff
@@ -371,24 +371,59 @@ function dosomething_reportback_form_submit($form, &$form_state) {
     $values['uid'] = $user->uid;
   }
 
-  // Save uploaded file.
-  if ($file = dosomething_reportback_form_save_file($form_state)) {
-    // Store new file's fid into values.
-    $values['fid'] = $file->fid;
-  }
+  if (module_exists('dosomething_rogue')) {
```

Comment:
ooo, so this module does exist & is enabled in prod currently. so I think we want to add an additional check here to not have this stuff happen in prod!